db/schema.rb

```diff
@@ -162,6 +164,26 @@
   add_index "repo_accesses", ["organization_id"], name: "index_repo_accesses_on_organization_id", using: :btree
   add_index "repo_accesses", ["user_id"], name: "index_repo_accesses_on_user_id", using: :btree
 
+  create_table "student_identifier_types", force: :cascade do |t|
```

Comment:
@dinever Looks like we don't have this table at this time, we might need to remove this